magento/pwa-studio
# [Doc] FAQ page

## Description

This PR creates a page of curated questions community member commonly ask about the project.

## Related Issue
Closes #PWA-545

## Acceptance 

Developers
Maintainers

### Verification Stakeholders

Developers
Maintainers

### Specification

### Verification Steps

1. Navigate to the `pwa-devdocs` project: `cd pwa-devdocs`
1. Run linter on new topic: `yarn lint src/frequently-asked-questions/index.md`
1. Verify no linting issues
1. Start the HTML preview server `yarn develop`
1. Verify site builds and runs
1. Navigate to the Getting Started section
1. Verify Frequently Asked Questions entry on the left nav
1. Click on the entry
1. Verify it loads the correct page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
